examples/app/server/app.py

```diff
@@ -15,7 +15,12 @@
 class PyTorchServer(PythonServer):
     def setup(self):
         self._model = torchvision.models.resnet18(pretrained=True)
-        self._device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
A similar comment here. The cloud compute specified in the app below does not support Apple Silicon hardware.